x/mint/types/constants.go

```diff
@@ -19,14 +19,8 @@ const (
 	TargetInflationRate  = 0.015
 )
 
-func InitialInflationRateAsDec() sdk.Dec {
-	return sdk.NewDecWithPrec(InitialInflationRate*1000, 3)
-}
-
-func DisinflationRateAsDec() sdk.Dec {
-	return sdk.NewDecWithPrec(DisinflationRate*1000, 3)
-}
-
-func TargetInflationRateAsDec() sdk.Dec {
-	return sdk.NewDecWithPrec(TargetInflationRate*1000, 3)
-}
+var (
+	InitialInflationRateAsDec = sdk.NewDecWithPrec(InitialInflationRate*1000, 3)
+	DisinflationRateAsDec     = sdk.NewDecWithPrec(DisinflationRate*1000, 3)
+	TargetInflationRateAsDec  = sdk.NewDecWithPrec(TargetInflationRate*1000, 3)
+)
```

Comment:
I don't care that much, but could we allocate once as an unexported global var, then use a function to enforce that they are only ever read from? global vars are the devil

Reply:
Resolved via dde51c8da0eeab38d96d58d359618c391a55a222